### ISSUE TITLE
1x2 and 2x2 not working properly

### ISSUE THREAD

**Issue by c0zmo** [ricardoquesada/vchar64#45]
I want to create a 1x2 font, where the lower part of the char is 64 chars away ("a"->"A"). This works for the first 64 chars, but the latter 64 chars are not editable. See attached screenshots.
<img width="763" alt="vchar64-1" src="https://user-images.githubusercontent.com/22218427/29493812-2bbd1e78-859e-11e7-8224-271d57a5558a.png">
<img width="594" alt="vchar64-2" src="https://user-images.githubusercontent.com/22218427/29493813-2da71bee-859e-11e7-858c-6d8d6b37d410.png">

**Comment by ricardoquesada** [comment]
interesting... could you attach the charset that you are trying to attach? thanks.

**Comment by c0zmo** [comment]
Yes, of course. Don't be confused by the strange patterns beginning at char 240 - those are the width information for each character, it is intended to be proportional in our project. :)

[1x2_Propo_v2.vchar64proj.zip](https://github.com/ricardoquesada/vchar64/files/1239241/1x2_Propo_v2.vchar64proj.zip)

**Comment by ricardoquesada** [comment]
so, the issue is that you used a `distance between chars = 64`. which is fine, but it works like this:
tile 0 = char0 + char 64
tile 1 = char 1 + char 65
tile 2 = char 2 + char 66
...
tile 62 = char 62 + char 126
tile 63 = char 63 + char 127

... but vchar will continue with this:

tile 64 = char 64 + char 128
tile 65 = char 65 + char 129
tile 66 = char 66 + char 130

as you can see, char 64, char 64, char 66 are being used more than once.
the easiest way to fix this is by using a "distance between chars = 128". (I attached the fixed version)

In any case, vchar should let you edit these "overflow" chars. if it doesn't probably it is a bug. but my suggestion is that you convert them to a "distance = 128".

I'll see how I can fix this in the future.

[1x2_Propo_v2_fixed.vchar64proj.zip](https://github.com/ricardoquesada/vchar64/files/1240760/1x2_Propo_v2_fixed.vchar64proj.zip)
